.test-env

```diff
@@ -1,6 +1,6 @@
 # Configs for testing repo download:
 SDK_TESTING_URL="https://github.com/algorand/algorand-sdk-testing"
-SDK_TESTING_BRANCH="V2"
+SDK_TESTING_BRANCH="improved-indexer-sync"
```

Comment:
```suggestion
SDK_TESTING_BRANCH="V2"
```
TODO: revert once https://github.com/algorand/algorand-sdk-testing/pull/309 is merged